presto-product-tests/conf/docker/singlenode-ldap-referrals/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '2'
+services:
+
+  presto-master:
+    image: 'prestodev/centos6-oj8-openldap-referrals:${DOCKER_IMAGES_VERSION}'
+    command: /docker/presto-product-tests/conf/docker/files/presto-launcher-wrapper.sh singlenode-ldap run
+    ports:
+      - '8443:8443'
+    volumes:
+      - ../../../conf/presto/etc/ldap-authenticator-referrals.properties:/docker/presto-product-tests/conf/presto/etc/password-authenticator.properties
+      - ../../../conf/presto/etc/access-control-ldap.properties:/docker/presto-product-tests/conf/presto/etc/access-control.properties
+      - ../../../conf/presto/etc/ldap-group-provider.properties:/docker/presto-product-tests/conf/presto/etc/group-provider.properties
+
+  application-runner:
+    image: 'prestodev/centos6-oj8-openldap-referrals:${DOCKER_IMAGES_VERSION}'
+    environment:
+      - TEMPTO_PROFILE_CONFIG_FILE=/docker/presto-product-tests/conf/tempto/tempto-configuration-for-docker-ldap.yaml
+      - CLI_ARGUMENTS=--server https://presto-master:8443 --keystore-path /etc/openldap/certs/coordinator.jks --keystore-password testldap
+
+  ldapserver:
+    image: 'prestodev/centos6-oj8-openldap-referrals:${DOCKER_IMAGES_VERSION}'
+    ports:
+      - '389:389'
+      - '636:636'
```

Comment:
Is there any point of exposing this ports? Is there anyone outside of docker cluster using these?

Reply:
I used these for debugging from host with ApacheDirectoryStudio, I will remove them

Reply:
Oh... then they are useful. I think it would be nice to write somewhere how these ports can be used for debugging with ApacheDirectoryStudio.

Reply:
I think we shouldn't expose them. If we need to debug LDAP locally, we can always modify to `docker-compose.xml` file. These are LDAP and LDAPS ports, it's pretty straighforward to do.